cdp_scrapers/legistar_utils.py

```diff
@@ -1310,6 +1313,99 @@ def get_content_uris(self, legistar_ev: Dict) -> List[ContentURIs]:
         )
         raise NotImplementedError
 
+    def inject_known_person(self, person: Person) -> Person:
+        """
+        Inject information from self.known_persons
+        if person exists in the dictionary for long-term static data
+
+        Parameters
+        ----------
+        person: Person
+            Person into which to inject data from known_persons
+
+        Returns
+        -------
+        Person
+            Input person updated with more information
+            or as-is if person not found in known_persons
+        """
+        try:
+            known_person = self.known_persons[person.name]
+        except (TypeError, KeyError):
+            return person
+
+        for attr in [
+            "email",
+            "is_active",
+            "name",
+            "phone",
+            "picture_uri",
+            "router_string",
+            "seat",
+            "website",
```

Comment:
Can you grab these attrs from the model definition? Instead of having a list of strings, like can we get a list of the required attrs for the ingestion model